### ISSUE TITLE
Warfront Completed Quest and Warfront Bosses reset time improve

### ISSUE THREAD

**Issue by spottsy** [SavedInstances/SavedInstances#240]
Thanks so much again for this addon been using for ages now. 

I notice there is another request for island and warfront reset request by another person but that was two months ago. 

Anyway if it it technically possible could Warfronts be added to this addon. 

I have one of each class a total of 12 Characters and I have to log on each one to check I haven't missed any. 

If it isn't possible to code this thanks again for what it does track it makes my life so much easier.

**Comment by SacredDuckwhale** [comment]
It's already being tracked, under "weekly quests". Yes, it technically isn't on a weekly reset, but there isn't a separate entry to avoid cluttering the UI with individual entries.

**Comment by LiangYuxuan** [comment]
The problem around here is that it is not possible to predict the time of this boss resets, when you kill a warfront boss during other faction's contributing stage. Warfront quests also have this problem.

But tracking the status of warfront might help SI to do this. Anyway, it works currently and I'm willing to improve this, but not so fast.

**Comment by spottsy** [comment]
Yeah you do a wonderful job with what you do track. I have noticed it glitching of late. I noticed that you have weekly quest tracked and if I mouse over the number 1 it shows me warfront. I have 12 x 120 characters one of each class. To be honest that is enough for me to use but it isn’t 100 percent reliable. I have done the warfront all this week so 12 times but only 10 show the warfront done with a 1 under each name and two not tracking it for some reason. Also the same with emesaries. If I do them on the same day it is ok but when the day ticks over it glitches. Maybe I have to reset I’ll try that this reset. 

 

AGAIN THANKS SO MUCH for all your hard work. It is valued and a lot of friends I suggest it to also love it. Saves me logging onto a toon to see if something is done. 

 

All the best
